README.md

```diff
@@ -9,11 +9,9 @@ Coming soon! This will be the home for the app for [Chain React 2023](https://cr
 1. `brew install fastlane`
 2. `yarn global add eas-cli@2.1.0` or `npm install -g eas-cli@2.1.0`
 
-### iOS Simulator
-
-1. `yarn ios`
+Note: After pulling `main`, if there are no changes to your `package.json` file, you don't need to run through the steps below, therwise follow the steps.
```

Comment:
`therwise` -> `otherwise`

But also I think you'll have to do these steps at least once if a fresh checkout. But yeah after pulling anytime after that as long as there are no package.json, app.json or config plugin changes they can be skipped.

Reply:
updated here: https://github.com/infinitered/ChainReactApp2023/pull/5/commits/b14936078ae9e6923a62453b22bf566348bf4a9f